### ISSUE TITLE
Tooltip does not work in a simple example

### ISSUE THREAD

**Issue by leolux** [tborychowski/perfecttooltip#9]
I am missing a simple example. Therefore I created one myself, but it does not work:
http://plnkr.co/edit/Nmq0UpzsNXeyjjJ6F2TY

Any idea why the tooltip does not show up?

**Comment by tborychowski** [comment]
a simple example is available under the main link: http://tborychowski.github.io/perfecttooltip/
as it is written there, to display a tooltip you need 2 things:
- target (a button)
- content (something to display in a tooltip)

In your example - you have the button, but you do not provide a text to display :-)

You can do it in multiple ways:
1. (recommended) add `data-title` to the button, e.g.
```html
<button class="btn btn-success" id="button8" data-title="tooltip content">position top, don't hide when hovering the tooltip</button>
```

2. add `title` attribute to the button, e.g.
```html
<button class="btn btn-success" id="button8" title="tooltip content">position top, don't hide when hovering the tooltip</button>
```
This way you have the fall-back (if the tooltip doesn't work or sth. - the browser native tooltip will).
However in some rare cases, some old browsers would display the title and the tooltip...

3. add `text` to the tooltip call, e.g.:
```js
$('#button8').tooltip({ position: 't', dontHideOnTooltipHover: true, text: 'tooltip content' });
```

**Comment by leolux** [comment]
Well, I copied the code for "button8" from http://tborychowski.github.io/perfecttooltip/. The problem seems to be a bug in the fierefox debugger tools because the "title" attribute is completely hidden. So I copied the HTML for "button8" using the debugger view of firefox which copies everything expect the title. I didn't expect such a trivial bug in firefox.

Of course, when adding the title everything works. Thank you for your support!
